cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/AbstractProgramRuntimeService.java

```diff
@@ -118,7 +118,7 @@ public final RuntimeInfo run(ProgramDescriptor programDescriptor, ProgramOptions
       return runtimeInfo;
     } catch (Exception e) {
       cleanUpTask.run();
-      LOG.error("Exception while trying to createPluginSnapshot", e);
+      LOG.error("Exception while trying to run program", e);
```

Comment:
I'd say let's remove? If its from an HTTP Handler, then it will be captured by [HttpExceptionHandler](https://github.com/caskdata/cdap/blob/develop/cdap-common/src/main/java/co/cask/cdap/common/HttpExceptionHandler.java#L49]?